google/cloud/internal/backoff_policy.h

```diff
@@ -101,7 +101,7 @@ class ExponentialBackoffPolicy : public BackoffPolicy {
    * @endcode
    *
    * @param initial_delay how long to wait after the first (unsuccessful)
-   *     operation.
+   *     operation and the minimum value for the delay between operations.
```

Comment:
optional: rename `initial_delay` to `minimum_delay`.

(It is fine to say: "I will do this in a later PR". FWIW, I think we will want it for #8755)